entity/contents/filters/gaussian_blur_filter_contents.cc

```diff
@@ -22,12 +22,19 @@ DirectionalGaussianBlurFilterContents::DirectionalGaussianBlurFilterContents() =
 DirectionalGaussianBlurFilterContents::
     ~DirectionalGaussianBlurFilterContents() = default;
 
-void DirectionalGaussianBlurFilterContents::SetBlurVector(Vector2 blur_vector) {
-  if (blur_vector.GetLengthSquared() < kEhCloseEnough) {
-    blur_vector_ = Vector2(0, kEhCloseEnough);
+void DirectionalGaussianBlurFilterContents::SetSigma(Sigma sigma) {
+  if (sigma.sigma < kEhCloseEnough) {
```

Comment:
Should this calculation be done in the Sigma constructor for safety? Or might there be different cutoffs for different situations?

Reply:
This cutoff is an implementation detail of the blur that's tied to the fragment shader. When the blur is set to 0, this makes the shader do 1 finite sample, which passes the image through with no blur (while retaining correct alpha mask behavior). We might do some different tricks in other filters that use gaussian distributions, so I'll leave this here for now.
I amended a comment detailing this behavior to make it clear that it more clear that it's implementation-specific.

Reply:
Hmm, post-merge thoughts: This check should be done on the post-transformed blur vector, otherwise blurs with sigma=0 of objects zoomed > ~500x will end up slightly blurred with more than 1 sample. Will do that in a follow-up.